osquery/config/parsers/tests/options_tests.cpp

```diff
@@ -30,4 +31,31 @@ TEST_F(OptionsConfigParserPluginTests, test_get_option) {
             true);
   c.reset();
 }
+
+TEST_F(OptionsConfigParserPluginTests, test_unknown_option) {
+  Config c;
+  std::map<std::string, std::string> update;
+
+  update["awesome"] =
+      "{\"options\": {\"fake\": 1, \"custom_fake\": 1, \"fake_custom_fake\": "
```

Comment:
I can't help but feel that you're missing a golden opportunity for something related to news being fake here.